### DISCUSSION TITLE
ER: Add Support for additional methods via WCv2

### DISCUSSION THREAD

**Discussion by ablankstory** [family/connectkit#272] [Ideas]
Apologies in advance, this issue is for the Family Wallet team - but I don't have access to the GitHub repos/issues for that product.

Loopring.io requires support for additional methods from WalletConnect v2  Without Family Wallet supporting these method, it can connect to the dApp but it can't much else. Here is a list:

"eth_accounts" > for connected account's address
"eth_chainId" > current chain
"eth_call" > check contact
"eth_getBalance" > for the Ethereum L1 balance
"eth_ecRecover" > unlocking the wallet
"personal_ecRecover" > unlocking the wallet

MetaMask has no problem connecting and unlocking. If Family isn't able to support this, we can advise users to import their wallet into MetaMask when looking to connect to our dApp. Obviously we would love to be able to allow them to connect directly with Family.

Thanks.

**Comment by ablankstory** [comment]
we updated the WCv2 implementation on our end to get around the issues and we're able to allow Family Wallet to function properly. Thanks.

**Reply by lochie** [reply]
Glad you found a resolve!
